geometry_msgs/Point center
float64 radius
float64 fit_rmse